TSOClient/tso.client/UI/Panels/EODs/UIBlackjackEOD.cs

```diff
@@ -0,0 +1,2451 @@
+            Player2Head.Visible = false;
+            Player2CardContainer.Visible = false;
+            Player2TotalBack.Visible = false;
+            Player2CardTotal.Visible = false;
+
+            playerPos3.Visible = false;
+            Player3Head.Visible = false;
+            Player3CardContainer.Visible = false;
+            Player3TotalBack.Visible = false;
+            Player3CardTotal.Visible = false;
+            
+            playerPos4.Visible = false;
+            Player4Head.Visible = false;
+            Player4CardContainer.Visible = false;
+            Player4TotalBack.Visible = false;
+            Player4CardTotal.Visible = false;
+            
+            DealerPos.Visible = false;
+            DealerHead.Visible = false;
+            DealerCardContainer.Visible = false;
+            DealerTotalBack.Visible = false;
+            DealerCardTotal.Visible = false;
+
+            Player1BetBack.Visible = false;
+            Player1BetAmount.Visible = false;
+            Player2BetBack.Visible = false;
+            Player2BetAmount.Visible = false;
+            Player3BetBack.Visible = false;
+            Player3BetAmount.Visible = false;
+            Player4BetBack.Visible = false;
+            Player4BetAmount.Visible = false;
+            DealerBetBack.Visible = false;
+            DealerBetAmount.Visible = false;
+            
+            int tempBalance;
+            int tempMinBet;
+            int tempMaxBet;
+            var split = balanceMinMaxBet.Split('%');
+            if (split.Length > 2)
+            {
+                if (Int32.TryParse(split[0], out tempBalance) & Int32.TryParse(split[1], out tempMinBet) & Int32.TryParse(split[2], out tempMaxBet))
+                {
+                    OwnerPanel = new UIManageEODObjectPanel(
+                        ManageEODObjectTypes.Blackjack, tempBalance, tempMaxBet * 6, 999999, tempMinBet, tempMaxBet);
+                    Add(OwnerPanel);
+                    // subscribe in order to send events based on type
+                    OwnerPanel.OnNewByteMessage += SendByteMessage;
+                    OwnerPanel.OnNewStringMessage += SendStringMessage;
+
+                    Controller.ShowEODMode(new EODLiveModeOpt
+                    {
+                        Buttons = 0,
+                        Height = EODHeight.Normal,
+                        Length = EODLength.Full,
+                        Tips = EODTextTips.Short,
+                        Timer = EODTimer.None,
+                        Expandable = false
+                    });
+
+                    SetTip(GameFacade.Strings["UIText", "259", "22"]); // "Closed for Maintenance"
+                }
+            }
+        }
+        /*
+         * Players supplied sound string
+         */
+        private void SoundHandler(string evt, string soundString)
+        {
+            // play the sound
+            HIT.HITVM.Get().PlaySoundEvent(soundString);
+            HIT.HITVM.Get().PlaySoundEvent(soundString);
```

Comment:
Why is this here twice? The second call will do nothing since two sounds of the same type cannot play at once.